src/core/less/scrolling.less

```diff
@@ -0,0 +1,28 @@
+.restrain-height() {
+  height: 100%;
+  overflow: hidden;
+}
+
+html.adapt-scrolling {
+  .restrain-height;
+
+  body, #app {
+    .restrain-height;
+  }
+
+  .navigation {
+    position: relative !important;
+  }
+
+  #app {
+    position: relative;
+    height: ~"calc(100% - @{navigation-height})";
+    overflow-y: scroll;
+    -webkit-overflow-scrolling: touch;
+  }
+
+  #wrapper {
+    padding: 0 !important;
```

Comment:
Just checking the need for `!important`s?

Reply:
Some themes will have #wrapper padding values at 64px or similar, this is just to force it to 0 for adapt-scrolling courses.

Reply:
worth adding comment (to the .less) to that effect?

Reply:
The specificity of the parent selector, ```html.adapt-scrolling```, should mean that you don't need to add the```!important``` declaration.

Agreed with @moloko that a comment outlining why the override is in place would help resolve any potential confusion.

Reply:
Done

Reply:
Would prefer to lose any unnecessary `!important`s since the selectors are already more than specific enough.